src/Services/TupleFilterService.php

```
@@ -108,12 +108,18 @@
     {
         $condition = $tuple->getCondition();
 
+        $conditionKey = '';
+
+        if ($condition instanceof ConditionInterface) {
+            $conditionKey = '#' . md5(json_encode($condition->jsonSerialize()));
```

PHPStan (2.1.15)

Parameter #1 $string of function md5 expects string, string|false given.


(argument.type)